src/components/Links/ContentHashLink.js

```diff
@@ -39,7 +39,7 @@ const ContentHashLink = ({ value, contentType }) => {
     return <DecodedError>{error}</DecodedError>
   }
   if (protocolType === 'ipfs') {
-    externalLink = `https://gateway.ipfs.io/ipfs/${decoded}`
+    externalLink = `https://${decoded}.ipfs.dweb.link` // using ipfs's secured origin gateway
```

Comment:
```suggestion
    externalLink = `https://dweb.link/ipfs/${decoded}` // using ipfs's secured origin gateway
```

We recommend keeping the path-based router for improved compatibility.
go-ipfs 0.5+ will redirect to correct subdomain, and go-ipfs 0.7+ will convert CID to DNS-compatible encoding, if needed